### PR TITLE
Csharp skeleton

### PR DESCRIPTION
**Pull request by chriszimmerman**
I created another test skeleton, this time for C#.

### CODE REVIEW

**Comment by mattmorgis** [comment]
🎉

**Review by mattmorgis — CHANGES_REQUESTED** [review]
Can you add a `README.md` for how to run this version?

File: csharp/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+#### Windows
+Run the script `run_unit_tests_windows.bat` in the `scripts` directory
+
+```bash
+$ cd scripts
+$ run_unit_tests_windows.bat 
+```
+
+#### Linux/OSX
```
Comment (mattmorgis) [inline csharp/README.md:24]: This can be run on macOS? 😮 

I'll give this a run through tomorrow and get it setup on my machine.

Comment (chriszimmerman) [reply csharp/README.md:24]: Yessir! Microsoft made .NET Core, as well as Visual Studio Community (Microsoft's IDE) available for Mac. 

I'd appreciate it if you did that and let me know if there are any fixes I need to make to the readme. 😃

Comment (mattmorgis) [reply csharp/README.md:24]: Apologies! Should be able to get to setting this up tonight

**Pull request closed without merging**